travis-ci/vmtest/mkrootfs.sh

```diff
@@ -87,7 +87,34 @@ packages=(
 
 pacstrap -C "$pacman_conf" -cGM "$root" "${packages[@]}"
 
+makedeps=(
+	# iproute2
+	asp
+	bison
+	fakeroot
+        flex
+	gcc
+        gettext
+	make
+	patch
+	pkgconf
+)
+
+pacman -Sy --noconfirm "${makedeps[@]}"
```

Comment:
Shouldn't `pacman` run in the `chroot` environment passed w/ `-r` option, e.g [0] ?
Without `-r` we're altering the build system, not the target fs.

`arch-chroot $root` should also work.

[0] https://git.archlinux.org/arch-install-scripts.git/tree/pacstrap.in#n115

Reply:
Will update.